airbyte-integrations/connectors/destination-elasticsearch/src/main/java/io/airbyte/integrations/destination/elasticsearch/SslUtils.java

```diff
@@ -0,0 +1,30 @@
+package io.airbyte.integrations.destination.elasticsearch;
+
+import java.io.ByteArrayInputStream;
+import java.security.KeyStore;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateFactory;
+import javax.net.ssl.SSLContext;
+import org.apache.http.ssl.SSLContextBuilder;
+import org.apache.http.ssl.SSLContexts;
+
+public class SslUtils {
+
+  public static SSLContext createContextFromCaCert(byte[] certAsBytes) {
```

Comment:
There already exists class `io.airbyte.db.util.SSLCertificateUtils`, I think this method belongs there as well

Reply:
Updated, thanks